invenio_github/errors.py

```diff
@@ -45,18 +45,29 @@ def __init__(self, user=None, repo=None, repo_id=None, message=None):
 class RepositoryDisabledError(GitHubError):
     """Repository access permissions error."""
 
-    message = u'This repository is not enabled for webhooks.'
+    message = "This repository is not enabled for webhooks."
 
     def __init__(self, repo=None, message=None):
         """Constructor."""
         super(RepositoryDisabledError, self).__init__(message or self.message)
         self.repo = repo
 
 
+class RepositoryNotFoundError(GitHubError):
+    """Repository not found error."""
+
+    message = "The repository does not exist."
+
+    def __init__(self, repo=None, message=None):
+        """Constructor."""
+        super(RepositoryNotFoundError, self).__init__(message or self.message)
```

Comment:
I think this is old way of subclassing in python. 


```suggestion
        super().__init__(message or self.message)
```

applies for all this file